config/config-template-long.yaml

```diff
@@ -89,8 +89,11 @@ scanners:
         executable: "zap.sh"  # for Linux
         # executable: "/Applications/OWASP ZAP.app/Contents/Java/zap.sh"    # for MacOS
 
-    # Defaults to False, set True do update ZAP's plugins
-    updateAddons: False
+    additionalOptions:
```

Comment:
can we move this to the bottom? (after the report option)

Reply:
done